clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/ops/CopyLastGoogleServerGroupAtomicOperation.groovy

```diff
@@ -217,14 +216,11 @@ class CopyLastGoogleServerGroupAtomicOperation implements AtomicOperation<Deploy
       }
     }
 
-    Autoscaler ancestorAutoscaler = GCEUtil.queryZonalAutoscaler(project,
-                                                                 Utils.getLocalName(ancestorServerGroup.zone),
-                                                                 ancestorServerGroup.name,
-                                                                 description.credentials)
-    BasicGoogleDeployDescription.AutoscalingPolicy ancestorAutoscalingPolicy =
-        ancestorAutoscaler ? GCEUtil.buildAutoscalingPolicyDescriptionFromAutoscalingPolicy(ancestorAutoscaler) : null
+    AutoscalingPolicy ancestorAutoscalingPolicy = ancestorServerGroup.autoscalingPolicy
+    BasicGoogleDeployDescription.AutoscalingPolicy ancestorAutoscalingPolicyDescription =
+      GCEUtil.buildAutoscalingPolicyDescriptionFromAutoscalingPolicy(ancestorAutoscalingPolicy)
 
-    newDescription.autoscalingPolicy = description.autoscalingPolicy ?: ancestorAutoscalingPolicy
+    newDescription.autoscalingPolicy = description.autoscalingPolicy ?: ancestorAutoscalingPolicyDescription
```

Comment:
Doesn't this mean we can't remove autoscaling from a server group during a clone once it's been added?